app/models/organization.rb

```diff
@@ -12,9 +12,10 @@ class Organization < ApplicationRecord
   has_many :uploads, through: :streams
   has_many :marc_records, through: :streams, inverse_of: :organization
   has_many :allowlisted_jwts, as: :resource, dependent: :delete_all
-  has_many :contact_emails, dependent: :delete_all
+  has_one :contact_email, dependent: :delete
   has_one_attached :icon
   has_many :statistics, dependent: :delete_all, as: :resource
+  accepts_nested_attributes_for :contact_email, update_only: true
```

Comment:
the result of this is that:
* if you create a new org, you must specify a single contact email
* if you edit the org details, you can change the email, but you can't remove it
* existing orgs without a contact email stay that way, until you add one

which i think is the behavior we want.

Reply:
I'm a bit wary of making this field a hard requirement to create an organization, but we can address that question as a separate issue.